### ISSUE TITLE
[Balance] Deafness and "paincrit"

### ISSUE THREAD

**Issue by Terror4000rus** [HaloSpaceStation/HaloSpaceStation13#2069]
Deafness lasts 5 minutes or so from one explosion. Reduce to 30 second or so?

There is no evident wounds which may get you in pain crit. I don't know how it works, but in one moment, even with tramadol and oxy you'll fall without possibility to get up.
I think it goes from some damage threshold, and if it is - its broken and really low.

**Comment by bloxgate** [comment]
Re deafness: see a medic; there are meds to reduce the time. It is a bit unlikely that the field medics will actually have it on them though, you may need to go to the actual infirmary.

For pain, painkillers aren't an instant fix-all. The most effective painkiller I'm aware of code-wise can only deal with up to 100 pain damage. The pain is based on all types of damage done, including toxin damage to organs inside you if your abdomen/head are hit. Full pain-crit occurs if your `shock_stage` is greater than 120. Pain killers can help here, as oxy reduces the values used to decide if `shock_stage` needs to increase by up to 200 - however it's my understanding that you can get very large values here quite easily if multiple limbs/more pain sensitive areas are hit. If you take greater than 30 pain damage after all painkiller modifiers you will start to move towards pain-crit. It will take some time for you to get out of paincrit once you stop the increase.

**TL;DR**: you need to treat all damage and give the painkillers time to work.

On our end, some solutions may involve adding ways to lower shock faster/making any existing ways more clear. On that note, have you tried using inaprovaline in addition to tramadol/oxy? I was under the impression it helped with getting people up faster

**Comment by BDpuffy420** [comment]
new medicine for ear damage was added
